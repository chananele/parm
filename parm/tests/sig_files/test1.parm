---
name: sig_file_test
exports:
  - test
method: find_single
pattern: |
  test: bne #0x1043c
...
