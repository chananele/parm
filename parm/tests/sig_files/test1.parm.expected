---
name: sig_file_test
result: pass
matches:
  test: 0x10458
...


